R/client-cloud.R

```diff
@@ -66,15 +66,39 @@ cloudClient <- function(service, authInfo) {
       listRequest(service, authInfo, path, query, "applications")
     },
 
-    getApplication = function(applicationId) {
-      path <- paste("/applications/", applicationId, sep = "")
-      application <- GET(service, authInfo, path)
+    getApplication = function(applicationId, dcfVersion) {
+      if (!(is.na(dcfVersion) || is.null(dcfVersion))) {
+        # On version >=1, applicationId refers to the id of the output, not the application.
+        path <- paste("/outputs/", applicationId, sep = "")
```

Comment:
Shouldn't we be checking the dcf version here?  Basically, what the comment is saying should be turned into an `if` statement.

Reply:
These if statements are equivalent, since there are no versions under 1.